clingox/ast.py

```diff
@@ -1391,3 +1391,70 @@ def theory_term_to_literal(x: AST, parse: bool = True) -> AST:
     if parse:
         x = clingo_literal_parser()(x)
     return _theory_term_to_literal(x, True, ast.Sign.NoSign)
+
+
+def normalize_symbolic_terms(x: AST):
+    """
+    Replaces all occurrences of objects of the class clingo.Function in an AST
+    by the corresponding object of the class ast.Function.
+
+    Parameters
+    ----------
+    x
+        The AST to rewrite.
+
+    Returns
+    -------
+    The rewritten AST.
+    """
+    return _NormalizeSymbolicTermTransformer().visit(x)
+
+
+def symbol_to_ast(x: clingo.Symbol, location: ast.Location) -> AST:
```

Comment:
I think we can make this a private function prefixing it with underscore. At least I can't imagine that this is something that would often be used in practice.